nvflare/job_config/fed_job.py

```diff
@@ -349,6 +383,91 @@ def _add_referenced_components(self, base_component, target):
                             # remove already added components from tracked components
                             self._components.pop(base_id)
 
+    def _add_script_executor(self, obj, target, tasks):
+        if obj._launch_external_process:
```

Comment:
Should we just make this `_add_executor_obj`?
Rename/merge `ScriptExecutor` with `ExecutorJobObj` since both of them are just wrappers?